src/model.js

```diff
@@ -310,6 +310,14 @@ export default class MediaPlayerObject {
         return this.callService(e, `${platform}_UNJOIN`, options);
       }
       if (platform === 'soundtouch') {
+        // if no entity is specified, remove self (leave group)
+        if (!entity || !entity.length) {
+          entity = this.id;
+        }
```

Comment:
I don't think we need to do this, should be handled already

https://github.com/kalkih/mini-media-player/blob/a3f56edfcdfb4cd6a0070fa1e43d90e76402dfeb/src/components/groupList.js#L49

Reply:
Alright, let me know, it's no biggie either way.